Australian Digital Observatory/content/the-ado-ecosystem/contents.lr

```diff
@@ -15,3 +17,45 @@ The ADO ecosystem aims to equip researchers with a set of modular, open source,
 This modular ecosystem approach gives flexibility, allowing researchers to pick and choose [resources](../resources) that are relevant and suitable for them, without being locked in to a specific stack (e.g., proprietary formats/tools, specific cloud infrastructure). The modular approach also recognises different entry levels in terms of researchers’ skills and needs. Furthermore, smaller tools are easier to maintain, and focusing on modularity and interoperability allows making use of and supporting existing tools and resources already in the community rather than reinventing the wheel.
 
 In designing and curating the ADO ecosystem, we take inspiration from the open source software community and [the Unix philosophy](https://medium.com/ingeniouslysimple/philosophy-of-unix-development-aa0104322491) of designing interoperable tools that each do one thing well, and from foundational tooling ecosystems such as [the R Tidyverse](https://rviews.rstudio.com/2017/06/08/what-is-the-tidyverse/#:~:text=The%20tidyverse%20is%20a%20coherent,being%20expanded%20by%20several%20contributors.), which has well demonstrated the suitability of such a structure to the academic research environment. We hope to learn from these communities, and also to contribute back to them.
+
+## What other resources are there for researchers?
+
+The ADO Ecosystem is a part of a broader community and ecosystem of open source or openly licensed resources and methods. We’d like to share a list of projects whose work we make use of in our own resources, and which we also frequently recommend to researchers to use either with our workflows or in their own right.
+The ADO Ecosystem is a part of a broader community and ecosystem of open source or openly licensed resources and methods. We’d like to share a list of projects whose work we make use of in our own resources, and which we also frequently recommend to researchers to use either with our workflows or in their own right.
+
+Data collection resources:
+
+-   [twarc](https://twarc-project.readthedocs.io/en/latest/ "https://twarc-project.readthedocs.io/en/latest/"): Command-line utility and Python library for collecting Twitter data    
```

Comment:
Suggestion: I think UX-wise, the best practice is to open external links in new tabs. (I haven't figured out how to do that in Markdown, so the only way is to use HTML apparently)

Reply:
Ah yes, that's a good point. I'm such a 'right click > open in new tab' person that I didn't even think to look at that 😸

Reply:
I'm the opposite - I've always felt it was best practice to not specify a link target and allow the user to control their own tabs? I find it annoying myself when something opens in a tab and I don't realise it. Maybe that's just me though?

Reply:
I've seen some sites have a little message that appears when you hover over a hyperlink that says 'opens in new tab' to let people know before they've clicked. Is it possible to do that?